utils/utils.go

```diff
@@ -171,6 +171,17 @@ func GetIdentifiers(args *skel.CmdArgs) (workloadID string, orchestratorID strin
 	return workloadID, orchestratorID, nil
 }
 
+func GetHandleID(netName string, containerID string, workload string) (string, error) {
+	handleID := fmt.Sprintf("%s.%s", netName, containerID)
+	log.WithFields(log.Fields{
+		"Network":     netName,
+		"ContainerID": containerID,
+		"Workload":    workload,
```

Comment:
Log is missing the handle itself :)

Reply:
Should that be part of the with fields or in the log message itself?

Reply:
fields